src/components/Info/Info.tsx

```diff
@@ -1,19 +1,22 @@
 import React, {ReactElement} from 'react'
 import Tooltip from '../Tooltip'
 import { Info as InfoSvg } from '../../svgs'
+import { EMPTY_STRING } from '../../consts'
+import classNames from 'classnames'
 
 import './info.scss'
 
 interface InfoProps {
   data: ReactElement | string
+  extraClass?: string
 }
-function Info({ data }: InfoProps) {
+function Info({ data, extraClass = EMPTY_STRING }: InfoProps) {
```

Comment:
no need to set the default value. classNames will remove any falsy variables

Reply:
thanks! didn't know that